Add paid shared nodes (#333)
chainstack/docs: docs/glossary/shared-node.md

A shared node is the node whose resources you share with other customers.

> With a shared node, you pay for JSON-RPC requests to the node and do not pay for the compute and storage resources used by the node.

Reviewer: ```suggestion
With a shared node, you only pay for JSON-RPC requests to the node.
```

Author: Same as above.
